node/node_utils.go

```diff
@@ -366,9 +366,8 @@ func (n *Node) SendIntervalDownlink(ctx context.Context, req IntervalRequest) (m
 	n.reconfigureMu.Lock()
 	if n.MinIntervalSeconds != 0 {
 		if n.MinIntervalSeconds > (req.IntervalMin * 60.0) {
-			n.reconfigureMu.Unlock()
-			return nil, fmt.Errorf(`requested uplink interval (%.2f minutes) exceeds the legal duty cycle limit of %.2f minutes,
-			increase the uplink interval`, req.IntervalMin, n.MinIntervalSeconds/60.0)
+			n.logger.Warnf(`requested uplink interval (%.2f minutes) exceeds the legal duty cycle limit of %.2f minutes,
```

Comment:
```suggestion
			// user preference in the EU is to warn the user of this limit, rather than prevent them from setting it at all.
			n.logger.Warnf(`requested uplink interval (%.2f minutes) exceeds the legal duty cycle limit of %.2f minutes,
```